Provide Data Privacy Documentation (#70)
zammad/zammad-admin-documentation: system/data-privacy.rst

      :align: center

      Click **Action > Delete**.

**in the “Manage > Users” Admin Panel**

   .. figure:: /images/system/data-privacy/delete-user-via-user-panel.gif
      :alt: Accessing the user deletion dialog under “Users” in the Admin Panel
      :align: center

      Use the **⋮ Actions** menu for the target user.

**in the “System > Data Privacy” Admin Panel**

   .. figure:: /images/system/data-privacy/delete-user-via-data-privacy-panel.gif
      :alt: Accessing the user deletion dialog under “Data Privacy” in the Admin Panel
      :align: center

      Use the **New Deletion Task** button. Search for users by name or email address.

.. hint:: 👥 **You can delete organizations, too.**

   If the customer you are deleting is the last user in their organization,
   a **Delete Organization?** option will be displayed in the user deletion dialog:

   .. figure:: /images/system/data-privacy/delete-organization-option.png
      :alt: Deleting an organization via the user deletion dialog
      :align: center
      :width: 50%

   (If you encounter an error when selecting “Yes” here,
   make sure there are no pending deletion tasks

>    for other customers from this organization.)

Author: I perosnally think that this text part is not needed.
The reason for this is that Zammad always only provides the organization removal screen if you are removing the last existing user. It does not take in account removal tasks.

I just double checked above.
I'd remove this text? I think you tried to adapt my note that you have to remove all organization members before adding a removal task for the last organization member?

Reviewer: Ah, I see what you mean now! How's this?

```suggestion
   (If this option does not appear,
   make sure there are no pending deletion tasks
   for other customers from this organization.)
```
